applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -111,10 +111,14 @@ static void stream_started_cb(struct bt_audio_stream *stream)
 {
 	int ret;
 
+	for (size_t i = 0U; i < ARRAY_SIZE(seq_num); i++) {
```

Comment:
Is this a new standard that we use `0U` instead of just `0` ?

Reply:
Use memset instead of loop

Reply:
About 0U: 
I actually thought about the same thing yesterday. I think the le_audio_(...) files use "size_t i = 0U" since they are based on Zephyr samples, which use this convention. Other files in the nef5340_audio application use "int i = 0". Maybe we should change the le_audio files to use "int i = 0"?